src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC <Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => (
+      <MovieCard movie={movie} />
+    ))}
```

Comment:
totaly agree with Piotr - each element of list shoud have key attribute , add it to <MovieCard> 😉